### ISSUE TITLE
RC Test: Molarity 1.5.0-rc.1

### ISSUE THREAD

**Issue by twant** [phetsims/QA#464]
@emily-phet, @zepumph, @jessegreenberg, @terracoda, @Matthew-phet molarity/1.5.0-rc.1 is ready for RC testing. Document issues in https://github.com/phetsims/molarity/issues and link to this issue.

Assigning to @ariel-phet for prioritization.

<details>
<summary><b>General RC Test</b></summary>

<h3>What to Test</h3>

- Click every single button.
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
- If there is sound, make sure it works.
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.
- Test all query parameters on all platforms. (See [QA Book](https://github.com/phetsims/QA/blob/master/doc/qa-book.md) for a list of query parameters.)
- Download HTML on Chrome and iOS.
- Make sure the iFrame version of the simulation is working as intended on all platforms.
- Make sure the XHTML version of the simulation is working as intended on all platforms.
- Complete the test matrix.
- Don't forget to make sure the sim works with Legends of Learning.
- Check [this](https://docs.google.com/spreadsheets/d/1umIAmhn89WN1nzcHKhYJcv-n3Oj6ps1wITc-CjWYytE/edit#gid=0) LoL spreadsheet and notify AR or AM if it not there.
- If this is rc.1 please do a memory test.
- When making an issue, check to see if it was in a previously published version
- If there is a console available, check for errors and include them in the Problem Description.
- As an RC begins and ends, check the sim repo. If there is a maintenance issue, check it and notify developers if there is a problem.

<h3>Link(s)</h3>

- **[Simulation](https://phet-dev.colorado.edu/html/molarity/1.5.0-rc.1/phet/molarity_all_phet.html)**
- **[iFrame](https://phet-dev.colorado.edu/html/molarity/1.5.0-rc.1/phet/molarity_all_iframe_phet.html)**
- **[XHTML](https://phet-dev.colorado.edu/html/molarity/1.5.0-rc.1/phet/xhtml/molarity_all.xhtml)**
- **[Test Matrix](https://docs.google.com/spreadsheets/d/1tb4vFUYbqCPPIDPOiV6BkamOZiZyWujxaF4al_2q4k4/edit#gid=1332044003)**
- **[Legends of Learning Harness](https://developers.legendsoflearning.com/public-harness/index.html?startPayload=%7B%22languageCode%22%3A%22en%22%7D)**

<hr>

</details>

<details>
<summary><b>Accessibility (a11y) RC Test</b></summary>

<h3>What to Test</h3>

- Specific instructions can be found below.
- Make sure the a11y feature doesn't negatively affect the sim in any way.
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
  - Test all keyboard navigation inputs (if applicable).
  - Test all forms of input with a screen reader (if applicable).

<h3>Focus and Special Instructions</h3>

- Please pay extra attention to the alerts around saturation point for various solutes (not all of them can be saturated). It can be helpful to use the shift-arrow navigation to move in small increments around saturation point. 
- The descriptions and alerts change based on whether the 'solution values' checkbox is checked, so please be sure to test the various features in both modes. 

<h3>Link(s)</h3>

- **[a11y View](https://phet-dev.colorado.edu/html/molarity/1.5.0-rc.1/phet/molarity_a11y_view.html)**
- **[Simulation](https://phet-dev.colorado.edu/html/molarity/1.5.0-rc.1/phet/molarity_all_phet.html)**
- **[Test Matrix](https://docs.google.com/spreadsheets/d/1tb4vFUYbqCPPIDPOiV6BkamOZiZyWujxaF4al_2q4k4/edit#gid=1332044003)**

<h3>Screen Readers</h3>

This sim supports screen readers. If you are unfamiliar with screen readers, please ask Katie to introduce you to screen readers. If you simply need a refresher on screen readers, please consult the [QA Book](link), which should have all of the information you need as well as a link to a screen reader tutorial made by Jesse. Otherwise, look over the a11y view before opening the simulation. Once you've done that, open the simulation and make sure alerts and descriptions work as intended.

<h3>Platforms and Screen Readers to Be Tested</h3>

- [ ] Windows 10 + Latest Chrome + Latest JAWS
- [ ] Windows 10 + Latest Firefox + Latest NVDA
- [ ] macOS + Safari + VoiceOver
- [ ] iOS + Safari + VoiceOver

<h3>Critical Screen Reader Information</h3>

We are tracking known screen reader bugs in [this Google Document](https://drive.google.com/open?id=12kTs-g-iKEIH1dyG7Q41_W_oNL4gUKbkW-IQgZjMUBw). If you find a screen reader bug, please check it against this list.

<h3>Keyboard Navigation</h3>

This sim supports keyboard navigation. Please make sure it works as intended on all platforms by itself and with a screen reader.

<h3>Final Requests</h3>

- [ ] If this sim is in [this list](https://github.com/phetsims/perennial/blob/master/data/accessibility), make an issue in [website](https://github.com/phetsims/website) to ask if PhET research page links need updating. Please assign to @terracoda and @emily-phet.

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>
  
  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary> 
  
  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>
  
  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>
  
  We typically assign the developer who opened the issue in the QA repository.

  </details>
  
  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>
  
  You should:

  1. Consult the [QA Book](https://github.com/phetsims/QA/blob/master/doc/qa-book.md).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by KatieWoe** [comment]
Memory test results:
![memsults](https://user-images.githubusercontent.com/41024075/71206169-24a22680-2261-11ea-9624-e68b05f82649.png)

**Comment by KatieWoe** [comment]
@twant, @emily-phet, @zepumph, @jessegreenberg, @terracoda, @Matthew-phet
Do we want new screenshots taken?

**Comment by zepumph** [comment]
I believe so, but we will want to tweak arrows over in https://github.com/phetsims/molarity/issues/201 first. Maybe next RC?

**Comment by KatieWoe** [comment]
Ok. Make an issue when you think its ready @zepumph

**Comment by zepumph** [comment]
Will do!

**Comment by KatieWoe** [comment]
QA is done

**Comment by zepumph** [comment]
Excellent thanks! Pinging @jessegreenberg so that he knows too.

**Comment by jessegreenberg** [comment]
Changes for this issue also need to be cherry picked for the RC

https://github.com/phetsims/chipper/issues/798
